integration_tests/conftest.py

```diff
@@ -36,6 +45,16 @@ def no_cleanup(request):
     return request.config.getoption("--no-cleanup")
 
 
+@pytest.fixture
+def base_branch(request):
+    return request.config.getoption("--base_branch")
```

Comment:
Can we name this `--base-branch` instead to have the same naming as for the existing `no-cleanup` option?

Reply:
Yes, changed it.